include/proxysql_glovars.hpp

```diff
@@ -88,6 +88,10 @@ class ProxySQL_GlobalVariables {
 		bool gdbg;
 		bool nostart;
 		bool my_monitor;
+		bool mysql_workers;
+		bool pgsql_workers;
+		bool mysql_admin;
+		bool pgsql_admin;
```

Comment:
nitpicking: we should follow a pattern, either first all mysql variables then all posgres variables (or viceversa), or follow some order: `*_monitor` , `*-workers` , `*_admin` .